### DISCUSSION TITLE
What is the minimum required Java version to build the SDK?

### DISCUSSION THREAD

**Discussion by kenfinnigan** [open-telemetry/opentelemetry-java#5465] [Q&A]
It's been a while since I last was involved with the project. so I was following the contribution guidelines and I wasn't able to build the project with Java 11 as per [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/CONTRIBUTING.md#requirements).

It gave me an error saying Java 17 was required.

Happy to submit a PR to change the contribution guide if it should be Java 17, but wanted to ask here first

**Comment by jack-berg** [comment]
Yes java 17 is required to build, but the artifacts produced by the build are java 8 compatible. Looks like we need to update the contribution guidelines. Sorry about that.

**Comment by kenfinnigan** [comment]
Thanks, I will submit a PR with the doc change
